### PR TITLE
Update 404 page

### LINKED ISSUE #600: Update 404 page to match figma page
As any user, if I visit a page that does not exist, I should be brought to the new 404 page.

### Mock/Design:

https://www.figma.com/file/tOK6AjkaRZ1NBXo191D64C/Sistering-%2F-User-Authentication-Process?node-id=980%3A5955

### PR DESCRIPTION
**Pull request by Boyazhang1**
## Ticket link

Closes #600

## Implementation description
* Updated the 404 page to match the new design on Figma

## Steps to test
1.Navigate to any invalid url, verify the following 404 is displayed 
Screen Shot 2022-08-15 at 5 30 20 PM

## What should reviewers focus on?
* Verify the implementation matches the figma (figma shown below)
Screen Shot 2022-08-15 at 5 31 21 PM

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/pages/admin/schedule/SchedulePostingPage.tsx [hunk 1]
```diff
@@ -421,9 +421,9 @@ const SchedulePostingPage = (): React.ReactElement => {
         duration: 10000,
         isClosable: true,
         variant: "subtle",
-        containerStyle: {
-          minWidth: "72vw",
-        },
+        // containerStyle: {
+        //   minWidth: "72vw",
+        // },
```
Comment (MatoPlus) [inline frontend/src/components/pages/admin/schedule/SchedulePostingPage.tsx:426]: : (

Comment (MatoPlus) [reply frontend/src/components/pages/admin/schedule/SchedulePostingPage.tsx:426]: was this causing an issue?

Comment (MatoPlus) [reply frontend/src/components/pages/admin/schedule/SchedulePostingPage.tsx:426]: Can we revert this change?

File: frontend/src/components/pages/NotFound.tsx [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
+      <FrownFace />
+      <Text textStyle="display-large" fontWeight="bold" my={8}>
+        Oh no! Page not found.
+      </Text>
+      <Text textStyle="caption" mb={2}>
+        Sorry, we can&apos;t find the page you&apos;re looking for.{" "}
+      </Text>
+      <Text textStyle="caption" mb={8}>
+        Try refreshing this page to reload or press the button below to return
+        to the main page.
```
Comment (sherryhli) [inline frontend/src/components/pages/NotFound.tsx:28]: Nit about the copy: not sure if refreshing is a helpful remediation on a 404 page. I think redirecting to main alone is enough

File: frontend/src/components/pages/admin/schedule/SchedulePostingPage.tsx (hunk 1 above)
Comment (Boyazhang1) [reply frontend/src/components/pages/admin/schedule/SchedulePostingPage.tsx]: It gave an error but not anymore lol when I just tried it, I reverted it back

File: frontend/src/components/pages/NotFound.tsx (hunk 2 above)
Comment (Boyazhang1) [reply frontend/src/components/pages/NotFound.tsx:28]: would this be better?
Screen Shot 2022-08-15 at 11 34 39 PM

Comment (sherryhli) [reply frontend/src/components/pages/NotFound.tsx]: This makes more sense to me. @MatoPlus @neelismail01 @a-twh are we okay with this copy change?

File: frontend/src/components/pages/NotFound.tsx [hunk 3]
```diff
@@ -1,11 +1,36 @@
...
+      <FrownFace />
+      <Text textStyle="display-large" fontWeight="bold" my={8}>
+        Oh no! Page not found.
+      </Text>
+      <Text textStyle="caption" mb={2}>
+        Sorry, we can&apos;t find the page you&apos;re looking for.{" "}
+      </Text>
+      <Text textStyle="caption" mb={8}>
+        Press the button below to return
+        to the main page.
```
Comment (sherryhli) [inline frontend/src/components/pages/NotFound.tsx:28]: Let's take care of this lint warning

File: frontend/src/components/pages/NotFound.tsx [hunk 4]
```diff
@@ -1,11 +1,36 @@
...
+      </Text>
+      <Button fontSize="16px" width="70%" onClick={() => history.push("/")}>
```
Comment (sherryhli) [inline frontend/src/components/pages/NotFound.tsx:30]: Nit: let's use the const defined `Routes.ts`

```suggestion
      <Button fontSize="16px" width="70%" onClick={() => history.push(HOME_PAGE)}>
```

File: frontend/src/components/pages/NotFound.tsx (hunk 2 above)
Comment (MatoPlus) [reply frontend/src/components/pages/NotFound.tsx]: +1

**Review by sherryhli — APPROVED** [review]
LGTM

**Pull request merged**